hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/node/TestDecommissionAndMaintenance.java

```diff
@@ -628,7 +629,7 @@ public void testMaintenanceEndsAutomaticallyAtTimeout()
     cluster.shutdownHddsDatanode(dn);
     waitForDnToReachHealthState(nm, dn, DEAD);
 
-    newEndTime = System.currentTimeMillis() / 1000 + 5;
+    newEndTime = Time.monotonicNow() / 1000 + 5;
```

Comment:
This also needs to be reverted (sorry, I meant [this comment](https://github.com/apache/ozone/pull/8096#discussion_r2035430828) for both usages in the same test).